### ISSUE TITLE
x/build: add LUCI plan9-amd64 builder

### ISSUE THREAD

**Issue by 0intro** [golang/go#63600]
- Hostname: plan9-amd64
- Certificate Request: [plan9-amd64.csr.txt](https://github.com/golang/go/files/12951050/plan9-amd64.csr.txt)

**Comment by cagedmantis** [comment]
@golang/release

**Comment by cagedmantis** [comment]
Here is the certificate: [plan9-amd64-1698251554.cert.txt](https://github.com/golang/go/files/13170338/plan9-amd64-1698251554.cert.txt)

**Comment by cagedmantis** [comment]
@0intro Have you had a chance to test out the bot with the certificate?

**Comment by 0intro** [comment]
@cagedmantis Yes. But since the bot can't run on Plan 9 directly (there is no support for Python on Plan 9), we would like to get some help to make changes to the bot, so it could be able to run on a Linux machine and send commands to a remote Plan 9 instance through SSH.

See #62025 and #63599.

**Comment by 0intro** [comment]
I've started the [plan9-amd64](https://chromium-swarm.appspot.com/bot?id=plan9-amd64) builder.
It's running, but there are still some issues. I'm working to fix them.
